python/torch_mlir_e2e_test/test_suite/slice_like.py

```diff
@@ -232,3 +232,75 @@ def SelectIntModule_basic(module, tu: TestUtils):
     module.forward(torch.randint(10, (5,5)))
 
 # ==============================================================================
+
+class SliceScatterModule(torch.nn.Module):
+    def __init__(self):
+        super().__init__()
+
+    @export
+    @annotate_args([
+        None,
+        ([-1, -1], torch.float32, True),
+        ([-1, -1], torch.float32, True),
+    ])
+    def forward(self, x, src):
+        return torch.ops.aten.slice_scatter(x, src, 1, 0, 1, 1)
```

Comment:
nit: add comments for arguments to improve readability.

Reply:
@gprateek93 , i think what was meant here was to do something like `slice_scatter(x, src, dim=0, start=0, end=1, step=1)`, so that it is clear to the reader what each argument means from just looking at the `forward` function